src/renderer/widgets/SendAssetModal/model/send-asset.ts

```diff
@@ -1,10 +1,47 @@
 import { createStore, createEffect, createEvent, sample, forward, attach } from 'effector';
+import { ApiPromise } from '@polkadot/api';
+import { BN } from '@polkadot/util';
 
-import { XcmConfig } from '@renderer/shared/api/xcm';
+import {
+  XcmConfig,
+  XcmTransfer,
+  getAvailableDirections,
+  AssetXCM,
+  getDestinationLocation,
+  getAssetLocation,
+  getAccountLocation,
+  estimateRequiredDestWeight,
+} from '@renderer/shared/api/xcm';
 import { xcmModel } from '@renderer/entities/xcm';
+import { Chain } from '@renderer/entities/chain';
+import { Asset } from '@renderer/entities/asset';
+import * as assetGuardModel from '@renderer/features/assets/AssetRouteGuard/model/asset-guard';
```

Comment:
As I see now, `asset-guard` model was not meant to be used outside of it's feature, so we need to remove it from reexports

Reply:
And declare a local event to indicate that flow has started